.github/workflows/main.yml

```diff
@@ -7,7 +7,75 @@ on:
     - cron:  '0 22 * * *'
 
 jobs:
-  Linux-build-and-test:
+  Lint:
```

Comment:
Since we are changing the names of these jobs, could we transform all of these names in to lower case?

Reply:
Sure.